mix.exs

```diff
@@ -36,10 +36,12 @@ defmodule Slime.Mixfile do
     [
       # HTML generation helpers
       {:phoenix_html, "~> 2.2", only: :test},
+      # Markdown processor
+      {:earmark, "~> 0.2.1", only: :test},
```

Comment:
I agree with @lpil, an optional dependency would be better.  If you can mock up a test that's sufficient for me @Rakoth 👍